packages/lib/src/components/internal/Button/Button.tsx

```diff
@@ -50,7 +50,7 @@ class Button extends Component<ButtonProps, ButtonState> {
             loading: <Spinner size="medium" />,
             redirect: (
                 <span className="adyen-checkout__button__content">
-                    <Spinner size="small" inline />
+                    <Spinner size="medium" inline />
```

Comment:
Does the design dictate a medium size spinner now?

Reply:
Yes - it is the one that is being used on our current designs